spec/models/project_spec.rb

```diff
@@ -192,8 +192,8 @@
     it 'does not send an email when user\'s receive_mailings attribute is set to false' do
       user = FactoryBot.create(:user, receive_mailings: false)
       project = FactoryBot.create(:project, user: user)
-      project.send_notification_to_project_creator(user)
-      assert ActionMailer::Base.deliveries.empty?
+      mail = project.send_notification_to_project_creator(user)
+      expect(mail).to be_nil
```

Comment:
yes it does return a Mailer object but since the 'receive_mailings' is set to false the object doesn’t get created and so the return value is nil. I tried to avoid using ActionMailer::Base again since it gives odd results.

Reply:
excellent thanks :-)